packages/blockchain-wallet-v4-frontend/src/modals/Swap/EnterAmount/Checkout/index.tsx

```diff
@@ -415,30 +427,104 @@ const Checkout: React.FC<InjectedFormProps<{}, Props> & Props> = (props) => {
             </GreyBlueCartridge>
           </MinMaxButtons>
         </Amounts>
-        <Button
-          nature='primary'
-          data-e2e='previewSwap'
-          type='submit'
-          jumbo
-          fullwidth
-          style={{ marginTop: '24px' }}
-          disabled={props.invalid || isQuoteFailed || disableInsufficientEth}
-        >
-          <FormattedMessage id='buttons.preview_swap' defaultMessage='Preview Swap' />
-        </Button>
+
+        {!showLimitError && !showError && (
+          <Button
+            nature='primary'
+            data-e2e='previewSwap'
+            type='submit'
+            jumbo
+            fullwidth
+            style={{ marginTop: '24px' }}
+            disabled={props.invalid || isQuoteFailed || disableInsufficientEth}
+          >
+            <FormattedMessage id='buttons.preview_swap' defaultMessage='Preview Swap' />
+          </Button>
+        )}
+
+        {!showLimitError && showError && (
+          <ButtonContainer>
+            <AlertButton>
+              {amtError === 'BELOW_MIN' ? (
+                <FormattedMessage
+                  id='copy.below_min'
+                  defaultMessage='{value} Minimum'
+                  values={{
+                    value:
+                      fix === 'FIAT'
+                        ? fiatToString({ unit: walletCurrency, value: fiatMin })
+                        : `${min} ${baseCoinfig.displaySymbol}`
+                  }}
+                />
+              ) : (
+                <FormattedMessage
+                  id='copy.above_max'
+                  defaultMessage='{value} Maximum'
+                  values={{
+                    value:
+                      fix === 'FIAT'
+                        ? fiatToString({ unit: walletCurrency, value: fiatMax })
+                        : `${max} ${baseCoinfig.displaySymbol}`
+                  }}
+                />
+              )}
+            </AlertButton>
+
+            <Text
+              size='14px'
+              color='textBlack'
+              weight={500}
+              style={{ marginTop: '24px', textAlign: 'center' }}
+            >
+              {amtError === 'BELOW_MIN' ? (
+                <FormattedMessage
+                  id='copy.swap_minimum_amount'
+                  defaultMessage='To avoid uncesssary fees and network slipage, the minimum amount for this pair is {amount}.'
+                  values={{
+                    amount:
+                      fix === 'FIAT'
+                        ? fiatToString({ unit: walletCurrency, value: fiatMin })
+                        : `${min} ${baseCoinfig.displaySymbol}`
+                  }}
+                />
+              ) : null}
```

Comment:
is a ternary needed?  what about just doing
```
{amtError === 'BELOW_MIN' && (....)
```

could also do below as well if you want but totally up to you as its just a small knitpick